src/applications/vaos/tests/e2e/vaos-cypress-helpers.js

```diff
@@ -644,8 +644,20 @@ export function initVAAppointmentMock({ cernerUser = false } = {}) {
   mockSubmitVAAppointment();
 }
 
-export function initVaccineAppointmentMock() {
+export function initVaccineAppointmentMock(unableToScheduleCovid = false) {
```

Comment:
I think for these helpers making the params an object makes the tests clearer. So like: `initVaccineAppointmentMock({ unableToScheduleCovid = false } = {})` and then when called it's `initVaccineAppointmentMock({unableToScheduleCovid: true })`.

Reply:
Interesting, yes that is much clearer to read.

Reply:
Suggestion added